src/Drupal/Driver/Cores/CoreInterface.php

```diff
@@ -216,4 +216,14 @@ public function configGet($name, $key = '');
    */
   public function configSet($name, $key, $value);
 
+  /**
+   * Create an entity.
+   */
+  public function entityCreate($entity_type, $entity);
+
+  /**
+   * Delete an entity.
+   */
+  public function entityDelete($entity_type, $entity);
```

Comment:
I don't see how to @param the delete function. The $entity argument can be of a number of different types, but code sniffer insists I specify a particular type and use it, so the tests fails if I don't. I tried:
   * @param string $entity_type
   *   Entity type machine name.
   * @param $entity
   *   The entity to delete, as an object with a property 'id' or a true entity.